test/parallel/test-tls-fast-writing.js

```diff
@@ -20,6 +20,7 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
+const fixtures = require('../common/fixtures');
 const common = require('../common');
```

Comment:
`common` should be the first thing `require()`'ed.

Reply:
I've fixed this